app/backend/src/couchers/metrics.py

```diff
@@ -11,11 +16,19 @@
 main_process_registry = CollectorRegistry()
 jobs_process_registry = CollectorRegistry()
 
+# ensures the directory used for prometheus multiprocessing exists and is clean
+multiproc_dir = pathlib.Path(config["PROMETHEUS_MULTIPROC_DIR"])
+if multiproc_dir.parent.exists():
+    shutil.rmtree(str(multiproc_dir), ignore_errors=True)
+    pathlib.Path(multiproc_dir).mkdir()
+    multiprocess.MultiProcessCollector(jobs_process_registry)
```

Comment:
Not a fan of this. `/tmp` won't work on all OSes. Is there any other way to do this other than through the filesystem? If not, maybe there's something convenient, OS-independent in the standard lib to create a temporary directory?

Reply:
I also don't like this. The problem is that this whole thing is necessary to have multiple separate processes talk to eachother. Because the processes don't have access to shared memory you need to set this var from something outside the process. 

I have checked it and it only works for me when I set an env var before starting the application. Even programatically setting an env var in the application doesn't do the trick. 

Because this whole functionality is not really relevant for local development I thought just not having metrics on jobs would be a decent solution to people running Windows.

Anyhow, tests are failing, so I need to do something about it anyway.

Reply:
Ignore what I said, figured out a way to do it by programatically setting an os env-var in an init function :)

Reply:
ARGH false alarm, doing it programtically didn't work. Spent the last two hours going into a whirlpool of madness on why this isn't working.

Reply:
Alright, found a way to manually clean up the http server. Still not 100% sure if this will fix the problem, since I cannot replicate the problem with a lingering server on port 8001 on my machine. The lingering http server is however a known issue that can occur, and the solution with manually killing the server on interrupt should work.